collect_app/src/androidTest/java/org/odk/collect/android/AllWidgetsFormTest.java

```diff
@@ -1104,7 +1104,7 @@ private String randomIntegerSeparator() {
     //endregion
 
     //region Custom TestRule.
-    private class FormEntryActivityTestRule extends IntentsTestRule<FormEntryActivity> {
+    private static class FormEntryActivityTestRule extends IntentsTestRule<FormEntryActivity> {
```

Comment:
Is those changes in Tests related to the update of gradle version?

Reply:
The build was failing because of these spotbugs detections. It is possible that these are linked to the updated gradle version